packages/react-components/src/components/VideoTile.tsx

```diff
@@ -192,12 +192,14 @@ const videoTileMoreMenuProps = {
 /* @conditional-compile-remove(pinned-participants) */
 const VideoTileMoreOptionsButton = (props: { contextualMenu?: IContextualMenuProps }): JSX.Element => {
   const { contextualMenu } = props;
+  const ids = useIdentifiers();
```

Comment:
is this a new hook that provides us with the data-ui-ids? is this context wrapping the composite internally? I haven't seen this yet? very cool!

Reply:
It is a context wrapping the composite internally and has been in our code for a while :). `_IdentifierProvider` is internal so I think it is ok to assign data-ui-id props directly as a prop to components but we should have a team discussion about it